data/common.yaml

```diff
@@ -13,9 +13,9 @@ bacula::director::postgresql::make_bacula_tables: ''
 bacula::director::db_type: 'pgsql'
 bacula::director::packages: []
 #bacula::ssl::ssl_dir: '/etc/puppetlabs/puppet/ssl'
-certfile: "%{hiera('bacula::conf_dir')}/ssl/${::clientcert}_cert.pem"
-keyfile: "%{conf_dir}/ssl/${::clientcert}_key.pem"
-cafile: "%{conf_dir}/ssl/ca.pem"
+bacula::certfile: "%{hiera('bacula::conf_dir')}/ssl/${::clientcert}_cert.pem"
+bacula::keyfile: "%{conf_dir}/ssl/${::clientcert}_key.pem"
+bacula::cafile: "%{conf_dir}/ssl/ca.pem"
```

Comment:
These look like they should be prefixed with `bacula::ssl::` rather than just `bacula::`, but they're not used anyway since https://github.com/xaque208/puppet-bacula/blob/master/manifests/ssl.pp#L24 has them commented out.  Would you mind adjusting those to reflect the class, or drop them entirely along with the class params on the ssl class?